### ISSUE TITLE
Compile error against Qt 5.12.1

### ISSUE THREAD

**Issue by nebulade** [CrimsonAS/gtkplatform#45]
On Archlinux using Qt 5.12.1 the build fails with:
```
qgtkwindow_render.cpp: In member function ‘void QGtkWindow::onWindowTickCallback()’:
qgtkwindow_render.cpp:124:40: error: ‘class QWindowPrivate’ has no member named ‘deliverUpdateRequest’
         QWindowPrivate::get(window())->deliverUpdateRequest();
                                        ^~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:794: .obj/qgtkwindow_render.o] Error 1
```

**Comment by rburchell** [comment]
bff59f87ba11cac1dfa710f021522372de7a776f in qtbase. Looks easy enough to patch, thanks for the report.

**Comment by rburchell** [comment]
Fixed by a9b84470ceefa481beef598c45bd7cae3042a61c I hope.
